test/core/test_ruby_errors.rb

```diff
@@ -119,7 +119,7 @@ def test_embedded_ruby1
 = unknown_ruby_method
 }
 
-    assert_ruby_error NameError,"(__TEMPLATE__):6", source
+    assert_ruby_error NameError,"(__TEMPLATE__):7", source
```

Comment:
This line is testing that backtrace is properly created, i.e. `unknown_ruby_method` is called on lineno 6 in `source`. Consider the case your code after 10 `:ruby` filters raised error. The error backtrace would say it raised an error on 10 lines after the actual location. It will make it hard to debug Slim template.

So this test should pass without modification. You need to generate another code to fix this problem.